src/main/generated/resources/data/minecolonies/crafterrecipes/sifter/diamond/dirt.json

```diff
@@ -6,12 +6,12 @@
       "item": "minecraft:dirt"
     },
     {
-      "item": "minecolonies:sifter_mesh_diamond"
+      "item": "minecolonies:sifter_mesh_diamond{Damage:0}"
     }
   ],
   "additional-output": [
     {
-      "item": "minecolonies:sifter_mesh_diamond"
+      "item": "minecolonies:sifter_mesh_diamond{Damage:0}"
```

Comment:
These changes weren't entirely intended, and I haven't specifically tested them as yet (but I don't expect them to cause issues since the sifter overrides things anyway).  It was a side effect of adding NBT support to `CustomRecipeBuilder`.